importer/EsEs.py

```diff
@@ -0,0 +1,135 @@
+        heritage = self.mapping["heritage"]["item"]
+        if self.has_non_empty_attribute("fecha"):
+            # 20 de febrero de 1985
+            months = ["enero", "febrero", "marzo",
+                      "abril", "mayo", "junio",
+                      "julio", "agosto",
+                      "septiembre", "octubre",
+                      "noviembre", "diciembre", ]
+            date_parts = self.fecha.split(" ")
+            if len(date_parts) != 5:
+                self.add_to_report("fecha", self.fecha)
+                self.add_statement("heritage_status", heritage)
+                return
+            else:
+                day = str(date_parts[0])
+                month = str(months.index(date_parts[2]) + 1)
+                year = str(date_parts[4])
+                date_string = "{}-{}-{}".format(year,
+                                                month.zfill(2), day.zfill(2))
+                try:
+                    date_dict = utils.date_to_dict(date_string, "%Y-%m-%d")
+                except ValueError:
+                    self.add_to_report("fecha", self.fecha)
+                    self.add_statement("heritage_status", heritage)
+                    return
+                qualifier = {"start_time": {"time_value": date_dict}}
+                self.add_statement("heritage_status", heritage, qualifier)
+        else:
+            self.add_statement("heritage_status", heritage)
+
+    def set_adm_location(self):
+        adm_q = None
+        municip_dic = self.data_files["municipalities"]
+        province_dic = self.data_files["provinces"]
+        if utils.count_wikilinks(self.municipio) == 1:
+            adm_q = utils.q_from_first_wikilink("es", self.municipio)
+        else:
+            # sometimes they're in ''
+            municip_raw = utils.remove_markup(self.municipio)
+            municip_match = [x for x in municip_dic if x[
+                "itemLabel"] == municip_raw]
+            if len(municip_match) == 1:
+                adm_q = municip_match[0]["item"]
+        if not adm_q:
+            province_raw = self.provincia_iso.lower()
+            prov_match = [x for x in province_dic if x[
+                "itemLabel"].lower() == province_raw]
+            if len(prov_match) == 1:
+                adm_q = prov_match[0]["item"]
+
+        if adm_q:
+            self.add_statement("located_adm", adm_q)
+        else:
+            self.add_to_report("municipio", self.municipio, "located_adm")
+
+    def set_special_is(self):
+        is_dic = self.data_files["tipobic"]["mappings"]
+        if self.has_non_empty_attribute("tipobic"):
+            special_raw = self.tipobic.lower()
+            special_match = [is_dic[x]
+                             for x in is_dic if x.lower() == special_raw]
+            if len(special_match) == 1:
+                self.remove_statement("is")
+                special_types = special_match[0]["items"]
+                for s in special_types:
```

Comment:
Is it possible to have more than one type in tipobic? If not I don't think it should be possible with multiple matches.

Reply:
Theoretically it can happen with lookup tables (you can have two q-items for a certain string). It doesn't seem to happen for this dataset, so I've removed it.